app/models/lead_provider.rb

```diff
@@ -2,6 +2,16 @@
 
 # This is actually ECFLeadProvider in all but name. See https://github.com/DFE-Digital/early-careers-framework/issues/698
 class LeadProvider < ApplicationRecord
+  ALL_PROVIDERS = [
```

Comment:
I wonder if this list can be in the seeds not in the main model? as we already define them there and this avoids confusion?